packages/foundation-models/README.md

```diff
@@ -17,9 +17,54 @@ $ npm install @sap-ai-sdk/foundation-models
   - Create a `.env` file in the sample-code directory.
   - Add an entry `AICORE_SERVICE_KEY='<content-of-service-key>'`.
 
-## Usage
+## OpenAI client
```

Comment:
[req] We just renamed it to `AzureOpenAIChatClient` and `AzureOpenAIEmbeddingClient`. In written text I would refer to it as "Azure OpenAI client".